lib/modules/us_core_guidance/clinicalnotes_sequence.rb

```diff
@@ -2,14 +2,24 @@
 
 module Inferno
   module Sequence
+    class ClinicalNoteAttachment
+      attr_reader :resource_class
+      attr_reader :attachment
+
+      def initialize(resource_class)
+        @resource_class = resource_class
+        @attachment = {}
+      end
+    end
+
     class USCoreR4ClinicalNotesSequence < SequenceBase
       group 'US Core R4 Profile Conformance'
 
       title 'Clinical Notes Guideline Tests'
 
-      description 'Verify that DocumentReference and DiagnosticReport resources on the FHIR server follow the US Core R4 Clinical Notes Guideline'
+      description 'Verify that DocumentReference and DiagnosticReport resources on the FHIR server follow the US Core Implementation Guide'
```

Comment:
We need to be filling this section out more going forward.  It should include our testing approach (we perform searches for each of the required clinical notes types for both document reference and diagnostic report, for example.) It should also describe what data needs to be on the server and how we validate it (we require that the selected patient has clinical notes of all types).